src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -242,6 +245,8 @@ bool debug_configuration::is_dumped_layer(const std::string& layerName, bool is_
         return true;
     if (dump_layers.empty() && dump_layers_result == 0)
         return true;
+    if (dump_layers.size() == 1 && dump_layers[0] == "*")
+        return true;
```

Comment:
You don't need this. If you set DumpLayersPath, without DumpLayers it will dump all layers because `dump_layers.empty() ==  True`

Reply:
Done.